apps/happy-blocks/block-library/universal-header/index.php

```diff
@@ -434,7 +418,7 @@
 		</div>
 	</div>
 </div>
-<div class="happy-blocks-mini-search happy-blocks-header__<?php echo esc_html( $args['active_tab'] ); ?>">
+<div class="happy-blocks-mini-search happy-blocks-header is-<?php echo esc_html( $happy_blocks_current_page ); ?>">
```

Comment:
Intentionally adding 'is-***' class to differentiate which page we are in on happy blocks context.